Terasology/DynamicCities
# Add city centre icon to minimap

**TODO**
- ~~Fix icon size variation on zoom~~
- ~~Make the icon persist~~

This PR adds an icon at the city centre. The icon stays on the map even when the city moves out of view, for easier navigation. 

**To Test:**
 - Start a game with DynamicCities enabled
 - Enter a city. The overlay is added to the minimap
 - An icon should be present at the centre of the city
- Now move around such that the city centre moves out of the area covered by the map. The icon should persist at the boundary of the map in the direction of the city centre.
